examples/plugins/README.md

```diff
@@ -1,39 +1,30 @@
 # Scope Probe Plugins
 
-Scope probe plugins let you insert your own custom metrics into Scope
-and get them displayed in the UI.
+Scope probe plugins let you insert your own custom data and controls into Scope and display them in the UI.
+The list of the current running plugins is displayed next to the label `PLUGINS` in the bottom right of the UI.
 
 <img src="../../imgs/plugin.png" width="800" alt="Scope Probe plugin screenshot" align="center">
 
-You can find some examples at the [the example
-plugins](https://github.com/weaveworks/scope/tree/master/examples/plugins)
-directory. We currently provide two examples:
-
-* A [Python
-  plugin](https://github.com/weaveworks/scope/tree/master/examples/plugins/http-requests)
-  using [bcc](http://iovisor.github.io/bcc/) to extract incoming HTTP
-  request rates per process, without any application-level
-  instrumentation requirements and negligible performance toll
-  (metrics are obtained in-kernel without any packet copying to
-  userspace).  **Note:** This plugin needs a [recent kernel version
-  with ebpf
-  support](https://github.com/iovisor/bcc/blob/master/INSTALL.md#kernel-configuration). It
-  will not compile on current [dlite](https://github.com/nlf/dlite)
-  and boot2docker hosts.
-* A [Go
-  plugin](https://github.com/weaveworks/scope/tree/master/examples/plugins/iowait),
-  using [iostat](https://en.wikipedia.org/wiki/Iostat) to provide
-  host-level CPU IO wait or idle metrics.
-
-The example plugins can be run by calling `make` in their directory.
-This will build the plugin, and immediately run it in the foreground.
-To run the plugin in the background, see the `Makefile` for examples
-of the `docker run ...` command.
-
-If the running plugin was picked up by Scope, you will see it in the
-list of `PLUGINS` in the bottom right of the UI.
-
-## Plugin ID
+
+## Official Plugins
+
+You can find all the official plugins at [Weaveworks Plugins](https://github.com/weaveworks-plugins).
+
+* [IOWait](https://github.com/weaveworks-plugins/scope-iowait): a Go plugin using [iostat](https://en.wikipedia.org/wiki/Iostat) to provide host-level CPU IO wait or idle metrics.
```

Comment:
I am not sure it's a good idea to have this here. We probably want it under `/site`. I am OK with it living here for now but maybe the listing (ant the rest of the documentation) should live on `/site` and we can link it from here.

@abuehrle thoughts?